tests/pre-commit/test.bats

```diff
@@ -39,6 +39,14 @@ repos:
       args:
         - --policy
         - examples/kubernetes/policy
+    # This hook is intended to change/fmt this file
+    - id: conftest-fmt
+      files: examples/kubernetes/deployment.yaml
+    - id: conftest-pull
+      args:
+        - --policy
+        - ./pulled-policies
```

Comment:
Please add a `.gitignore` for this directory.

Reply:
@jalseth - I can add this, but this is being cloned/pulled as part of a temporary directory - as part of testing:

https://github.com/open-policy-agent/conftest/blob/58ace24522e68fe7c603ad645c4b2fbeb5ddecdb/tests/pre-commit/test.bats#L12-L29

Thoughts?